app/views/shared/_troubleshooting_options.html.erb

```diff
@@ -2,7 +2,7 @@
 locals:
 * heading_tag: Tag name for heading. Does not affect visual appearance. Defaults to :h2.
 * heading: Heading text.
-* options: List of link options to display, as an array of hashes with `url`, `text`, `external` values.
+* options: List of link options to display, as an array of hashes with `url`, `text`, `new_tab` values.
```

Comment:
Whoops 😅  Thanks for fixing.

Interestingly, for reasons I cannot recall, we call it both things in the JSX implementation 🤔 

https://github.com/18F/identity-idp/blob/12f214fdb50dc2af4b819dff7fc9e820ca90e106/app/javascript/packages/components/troubleshooting-options.jsx#L34-L36